test/src/test/java/jenkins/model/JenkinsManagePermissionTest.java

```diff
@@ -207,15 +219,40 @@ public void globalConfigAllowedWithManagePermission() throws Exception {
 
         HtmlForm form = j.createWebClient().goTo("configure").getFormByName("config");
         HtmlPage updated = j.submit(form);
-        assertEquals("User with Jenkins.MANAGE permission should be able to update global configuration",
-                     200, updated.getWebResponse().getStatusCode());
+        assertThat("User with Jenkins.MANAGE permission should be able to update global configuration",
+                     updated.getWebResponse(), hasResponseCode(HttpURLConnection.HTTP_OK));
     }
 
     private String getShell() {
         Descriptor descriptorByName = j.getInstance().getDescriptorByName("hudson.tasks.Shell");
         return ((Shell.DescriptorImpl) descriptorByName).getShell();
     }
 
+    private Matcher<WebResponse> hasResponseCode(final int httpStatus) {
```

Comment:
NIT: just a function of style but possibly more reusable in the future as a static class.